NoteFind/Views/NoteCell.h

```diff
@@ -12,13 +12,17 @@
 #import "Parse/PFImageView.h"
 
 NS_ASSUME_NONNULL_BEGIN
+@protocol DataFailureDelegate
+- (void)showErrorMessage;
```

Comment:
Nice job setting up the communication between the table view cell and table view itself!

Reply:
thank you!